test/fail_compilation/test11176.d

```diff
@@ -1,5 +1,5 @@
 /*
-REQUIRED_ARGS: -de
+REQUIRED_ARGS:
 ---
 fail_compilation/test11176.d(12): Deprecation: b.ptr cannot be used in @safe code, use &b[0] instead
 fail_compilation/test11176.d(16): Deprecation: b.ptr cannot be used in @safe code, use &b[0] instead
```

Comment:
This isn't making any sense to me.  Why are the tests passing?  The output of this test should be
```
Error: `b.ptr` cannot be used in `@safe` code, use `&b[0]` instead
```

Reply:
Excellent catch - the test helped me to realize that there was another Deprecation message that needed maintenance (i.e. the one for dynamic arrays).